src/template/wasm2c.declarations.c

```diff
@@ -444,4 +444,85 @@ static float wasm_sqrtf(float x) {
   return sqrtf(x);
 }
 
+static inline void memory_fill(wasm_rt_memory_t* mem, u32 d, u32 val, u32 n) {
```

Comment:
Would it make sense to somehow only emit this code when bulk memory is enabled?  Perhaps we could define `_WASM2C_HAS_BULK_MEMORY` and make this conditional?  I guess it doesn't make much difference?  Maybe tiny modules would compile marginally faster? 

Maybe not worth it?

Reply:
We can do this if you want, but given that bulk memory is enabled by default (and has been for almost a year), it seems like a niche case... and it's not like these are particularly complicated functions to compile. (Some of the math functions are arguably more complicated and we emit those unconditionally...)